lib/constellation.rb

```diff
@@ -0,0 +1,39 @@
+class ConstellationHttpException < StandardError
+end
+
+require 'net/http'
+module Constellation
+  def self.create_rsvp!(rsvp_params)
+    return unless constellation_enabled?
+    uri = URI.parse("#{app}/rsvps")
+    http = Net::HTTP.new(uri.host, uri.port)
+    unless Rails.env.development?
+      http.use_ssl = true
+      http.verify_mode = OpenSSL::SSL::VERIFY_NONE
```

Comment:
I wasn't able to test this since I'm just running this app locally -> Constellation locally.

Once https://github.com/artsy/constellation/pull/27 is merged I can test against staging (w/ SSL) and have it more realistic. This is just a placeholder.